tika-core/src/test/java/org/apache/tika/TikaIT.java

```diff
@@ -27,8 +27,8 @@
     public void testToString() {
         String version = new Tika().toString();
         assertNotNull(version);
-        assertTrue(version.matches(
-                "Apache Tika \\d+\\.\\d+\\.\\d+(-(?:ALPHA|BETA))?(?:-SNAPSHOT)?"));
+//        assertTrue(version.matches(
```

Comment:
I would keep this .. as the issues is not here.. but in Tika().toString() - it only returns "Apache Tika" (to get a version it needs a pom.properties file - but that does not exist